Signal/src/UserInterface/Strings.swift

```diff
@@ -15,16 +15,13 @@ import Foundation
 
```

Comment:
IMO we should try to duplicate strings less and move all common usage of a string to this file, in which case a namespace like CallStrings is arguably useful. We just haven't yet de-duped anything besides call strings.